rca/programmes/models.py

```diff
@@ -380,6 +380,12 @@ class ProgrammePage(TapMixin, ContactFieldsMixin, BasePage):
 
     next_open_day_date = models.DateField(blank=True, null=True)
     link_to_open_days = models.URLField(blank=True)
+    book_or_view_all_open_days_link_title = models.CharField(
+        max_length=255,
+        blank=True,
```

Comment:
I think it's worth adding validation for this to be required if the link_to_open_days is set. At the moment you could remove the default text and have a blank link.

Reply:
Thanks! Added in 03b44ba6

<details><summary>Screenshot</summary>

[![Screenshot](https://github.com/torchbox/rca-wagtail-2019/assets/7713776/7a97a2fe-e619-4374-80c1-bad24e691493)](https://github.com/torchbox/rca-wagtail-2019/assets/7713776/7a97a2fe-e619-4374-80c1-bad24e691493)


</details>